app/src/main/res/values-es-rAR/strings.xml

```diff
@@ -715,4 +730,66 @@
+    <string name="etp_cookies_description">Bloquea las cookies que utilizan las redes publicitarias y las empresas de análisis para compilar tus datos de navegación en muchos sitios.</string>
+    <!-- Category of trackers (cryptominers) that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_cryptominers_title">Criptomineros</string>
+    <!-- Description of cryptominers that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_cryptominers_description">Impide que los scripts maliciosos obtengan acceso a tu dispositivo para extraer moneda digital.</string>
+    <!-- Category of trackers (fingerprinters) that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_fingerprinters_title">Identificadores de huellas digitales</string>
+    <!-- Description of fingerprinters that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_fingerprinters_description">Impide que se recopilen datos identificables acerca de tu dispositivo que pueden usarse para fines de rastreo.</string>
+    <!-- Category of trackers (tracking content) that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_tracking_content_title">Contenido de rastreo</string>
+    <!-- Description of tracking content that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_tracking_content_description">Detiene la carga de anuncios externos, videos y otro contenido que contiene código de rastreo. Puede afectar la funcionalidad del sitio web.</string>
+    <!-- Enhanced Tracking Protection Onboarding Message shown in a dialog above the toolbar. The first parameter is the name of the application (For example: Fenix) -->
+    <string name="etp_onboarding_message">Cuando el escudo cambia de color, %s está bloqueando los rastreadores en un sitio web. Tocalo para ver qué es lo que está bloqueando.</string>
+    <!-- Enhanced Tracking Protection message that protection is currently on for this site -->
+    <string name="etp_panel_on">Las protecciones están habilitadas para este sitio</string>
+    <!-- Enhanced Tracking Protection message that protection is currently off for this site -->
+    <string name="etp_panel_off">Las protecciones están OFF para este sitio</string>
+    <!-- Header for exceptions list for which sites enhanced tracking protection is always off -->
+    <string name="enhanced_tracking_protection_exceptions">La protección contra rastreo mejorada está deshabilitada para estos sitios</string>
+    <!-- About page Your rights link text -->
+    <string name="about_your_rights">Tus derechos</string>
+    <!-- About page link text to open open source licenses screen -->
+    <string name="about_open_source_licenses">Bibliotecas de código abierto que utilizamos</string>
+    <!-- Open source licenses page title
+    The first parameter is the app name -->
+    <string name="open_source_licenses_title">% s | Bibliotecas OSS</string>
```

Comment:
Looks like there is a space here in the placeholder?

Reply:
Updated in Pontoon, thanks!